extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/widgets/LogoWidget.kt

```diff
@@ -27,7 +27,7 @@ class LogoWidget constructor(
   ),
   marginX: Float = 20f,
   marginY: Float = 20f,
-) : BitmapWidget(
+) : RecyclingBitmapWidget(
```

Comment:
```suggestion
) : BitmapWidget(
```

Implementations in `androidauto` will need to use the upgradable object.

We can adopt the new api when it becomes available but it is going to be locked on [10.5](https://github.com/mapbox/mapbox-maps-android/pull/1706), it'll depend stable release versions when they become available

Reply:
We won't use the `RecyclingBitmapWidget` until we upgrade the androidauto extension minimum version to a version that includes the new class.

Another option is to backport this to all the releases (for example, `10.5.1`, `10.6.1`, `10.7.1`, `10.8.1`). That is an obvious headache that would require a major issue, and I think we all agree this issue does not qualify for that.

So our other option is to just remove  `RecyclingBitmapWidget` from the AA changes. We can use it becomes available. Developers have the option to upgrade and implement it themselves.